webui/react/src/components/ExperimentTable.tsx

```diff
@@ -0,0 +1,92 @@
+import { Table } from 'antd';
+import { ColumnsType, ColumnType } from 'antd/lib/table';
+import React, { MouseEventHandler } from 'react';
+
+import Badge from 'components/Badge';
+import linkCss from 'components/Link.module.scss';
+import { actionsColumn, Renderer, startTimeColumn, stateColumn,
+  userColumn } from 'table/columns';
+import { CommonProps, Experiment } from 'types';
+import { alphanumericSorter } from 'utils/data';
+import { floatToPercent } from 'utils/string';
+import { experimentDuration, shortEnglishHumannizer } from 'utils/time';
+import { experimentToTask } from 'utils/types';
+
+import css from './ExperimentTable.module.scss';
+import ProgressBar from './ProgressBar';
+import { tableRowClickHandler } from './TaskTable';
+
+interface Props extends CommonProps {
+  experiments?: Experiment[];
+}
+
+const progressRenderer: Renderer<Experiment> = (_, record) => {
+  if (!record.progress) return;
+  return <ProgressBar
+    percent={record.progress * 100}
+    state={record.state}
+    title={floatToPercent(record.progress, 0)} />;
+};
+
+const durationRenderer: Renderer<Experiment> = (_, record) => {
+  return shortEnglishHumannizer(experimentDuration(record));
+};
+
+const descriptionRenderer: Renderer<Experiment> = (_, record) => {
+  // TODO handle displaying labels not fitting the column width
```

Comment:
👍  We can apply wrapping to the label containing div below.

Reply:
I can apply that but probably we might need a more involved solution https://determinedai.atlassian.net/browse/DET-3398